src/unix/udp.c

```diff
@@ -507,6 +507,32 @@ static int uv__set_reuse(int fd) {
   return 0;
 }
 
+/*
+ * The Linux kernel suppresses some ICMP error messages by default for UDP
+ * sockets. Setting IP_RECVERR/IPV6_RECVERR on the socket enables full ICMP
+ * error reporting, hopefully resulting in faster failover to working name
+ * servers.
+ */
+
+static int uv__set_recverr(int fd) {
+#if defined(IP_RECVERR) && defined(IPV6_RECVERR)
+  int yes;
+  yes = 1;
+  struct sockaddr_in sockfd;
+  unsigned int sockfd_len = sizeof(sockfd);
```

Comment:
You could use `uv__sockaddr` so it works for both ipv4 and ipv6.

Reply:
Both fixed by adding second sa_family_t argument to the function.